cli/types/cypress.d.ts

```diff
@@ -5676,10 +5716,13 @@ declare namespace Cypress {
   }
 
   interface EnqueuedCommand {
+    id: string
     name: string
     args: any[]
     type: string
     chainerId: string
+    injected: boolean
```

Comment:
Do we want to expose these types to users?

Reply:
I don't think it hurts to do so.